### ISSUE TITLE
Subpar results on 12 hours of male speech both when pretrained and from scratch

### ISSUE THREAD

**Issue by patdflynn** [NVIDIA/tacotron2#408]
Having issues training tacotron on 12 hours of male speech. The model has been trained from scratch a few times and we had thought failures to converge thus far were from the noisiness of the data, and have tried several methods of cleaning.

Here's are attention plots for the model trained from scratch.
3000 epochs:
![image](https://user-images.githubusercontent.com/44388720/90079712-836b5300-dcbd-11ea-9e86-afa2976f00d5.png)

13000 epochs:
![image](https://user-images.githubusercontent.com/44388720/90079724-8f571500-dcbd-11ea-9949-ce67923802b0.png)

We also tried using a tacotron pretrained on LJSpeech but the model got worse as it trained. After a few thousand epochs, only the first word of each sentence was intelligible.

Alignment plot for the pretrained model after a few epochs on our data:
![image](https://user-images.githubusercontent.com/44388720/90079556-19eb4480-dcbd-11ea-9a41-4db5393d5dc8.png)

And after two thousand more epochs on our data the alignment collapses:
![image](https://user-images.githubusercontent.com/44388720/90079665-6040a380-dcbd-11ea-92bc-f6a63b3461d7.png)
![image](https://user-images.githubusercontent.com/44388720/90079674-66cf1b00-dcbd-11ea-8212-666f97fb02b6.png)

**Comment by kannadaraj** [comment]
@patdflynn i am facing this issue too.. Did you find any solutions? Thanks..

**Comment by youmebangbang** [comment]
Do you have silence trimmed from front and end of each wav, as well as extra .5 seconds of silence added to end of each wav? Are you using a terminating character at the end of each line of text (such as '~'). Is your audio normalized? How accurate is your text?

**Comment by patdflynn** [comment]
@kannadaraj Sorry for the delay. We've only just solved this by copying and warping/pitching our dataset along the lines of [this post](https://github.com/keithito/tacotron/issues/243). Decent results around 30k epochs.

**Comment by patdflynn** [comment]
@youmebangbang Trying another round of training along these lines.
